TsJournalTrello.gs

```diff
@@ -0,0 +1,361 @@
+
+// Private Functions
+// =================
+
+// General
+// -------
+
+/**
+ * All external function calls should call this to ensure standard 
+ * processing - logging, errors, etc - is always done.
+ *
+ * @param {Array} config:
+ *   [0] {Function} prefunction
+ *   [1] {String} eventName
+ *   [2] {String} onErrorMessage
+ *   [3] {Function} mainFunction
+ *
+ * @param {Object}   args       The argument passed to the top-level event handler
+ */
+
+function eventHandler_(config, args) {
+
+  try {
+
+   var userEmail = Session.getActiveUser().getEmail()
+   var logSheetId = PropertiesService.getScriptProperties().getProperty("LOG_SHEET_ID")
+   
+   Log_ = BBLog.getLog({
+     level:                DEBUG_LOG_LEVEL_, 
+     displayFunctionNames: DEBUG_LOG_DISPLAY_FUNCTION_NAMES_,
+     sheetId:              logSheetId, //'1F17gbA3digJWgTM-qFI5sFDjmstzG3NwP4YKCheSfiA',
+   })
+    
+   Log_.info('Handling ' + config[0] + ' from ' + (userEmail || 'unknown email') + ' (' + SCRIPT_NAME + ' ' + SCRIPT_VERSION + ')')
+    
+   // Call the main function
+   return config[2](args)
+    
+  } catch (error) {
+
+  var handleError = Assert.HandleError.DISPLAY_FULL
+
+  if (!PRODUCTION_VERSION_) {
+    handleError = Assert.HandleError.THROW
+  }
+
+  var assertConfig = {
+    error:          error,
+    userMessage:    config[1],
+    log:            Log_,
+    handleError:    handleError, 
+    sendErrorEmail: SEND_ERROR_EMAIL_, 
+    emailAddress:   ADMIN_EMAIL_ADDRESS_,
+    scriptName:     SCRIPT_NAME,
+    scriptVersion:  SCRIPT_VERSION,
+    }
+
+    Assert.handleError(assertConfig)
+    }
+    
+    } // eventHandler_()
+
+// Private event handlers
+// ----------------------
+
+/**
+ *
+ *
+ * @param {object} 
+ *
+ * @return {object}
+ */
+ 
+function linkHeaders_(DocId) {
+
+  Log_.functionEntryPoint()
+  
+  //Open the organisation spreadsheet
+  var orgData = SpreadsheetApp.openById(ORG_ID_)
+      .getSheetByName('Organisations')
+      .getDataRange()
+  
+  var data = orgData.getValues()
+  var timesheetUrl = null
+  var trelloBoardUrl = null
+  var trelloBoardId = null
+
+  // Get the organisation name from the journal name, text before '- journal' 
+  var str = ' - Journal'
+  var CHAR_TO_END_ = str.length
+  var orgName = DocumentApp.getActiveDocument().getName().slice(0, -CHAR_TO_END_)
+  var orgFound = false
+    
+  //Loop through the rows of data
+  orgFound = data.some(function(row)  {
+      
+    //check if the company name matches the orgName
+    var orgNameSearch = row[ORG_NAME_COL_]
+      
+    if (orgNameSearch === orgName) {
+    
+      //Get the timesheet and Trello Board Url
+      timesheetUrl = row[ORG_TS_COL_]
+      trelloBoardUrl = row[TRELLO_COL_]
+      return true
+      
+      }
+      
+  })   
+    
+  if (!orgFound) {
+    
+    Log_.warning('Organisation Name: ' + orgName + ' not found in Org Spreadsheet')
+    return
+      
+  } else {
```

Comment:
Redundant else. The script will only get past here if condition past so no need to indent.

```suggestion
  if (!orgFound) {
    Log_.warning('Organisation Name: ' + orgName + ' not found in Org Spreadsheet')
    return
  }
 Log_.info('Organisation Name: ' + orgName + ' found in Org Spreadsheet')
```